editor/src/components/canvas/padding-utils.ts

```diff
@@ -5,48 +5,71 @@ import { ElementInstanceMetadataMap, isJSXElement } from '../../core/shared/elem
 import { CanvasVector } from '../../core/shared/math-utils'
 import { ElementPath } from '../../core/shared/project-file-types'
 import { assertNever } from '../../core/shared/utils'
-import { CSSNumber, CSSNumberUnit, CSSPadding } from '../inspector/common/css-utils'
+import { CSSNumber, CSSPadding } from '../inspector/common/css-utils'
 import { EdgePiece } from './canvas-types'
 
 type CSSPaddingKey = keyof CSSPadding
-export type SimpleCSSPadding = { [key in CSSPaddingKey]: number }
+type CSSPaddingMappedValues<T> = { [key in CSSPaddingKey]: T }
+export type CSSPaddingMeasurements = CSSPaddingMappedValues<PaddingMeasurement>
+
+export interface PaddingMeasurement {
+  value: CSSNumber
+  renderedDimension: number
```

Comment:
Please can you rename this to something like `renderedValue` or `renderedValuePx`? It took me a while to figure out what this value represented and why we were dividing by it later on

Reply:
done!